src/test/java/ChangePasswordTest.java

```diff
@@ -15,19 +15,24 @@
 
     private String randomEmail;
 
-    @Test(priority = 1)
+    @BeforeMethod
     public void registerUserWithValidParameters() {
         new Navigation().navigateToUrl(BASE_URL.getValue());
         MainPageBL mainPageBL = new MainPageBL();
-        RegisterPageBL registerPageBL = mainPageBL.getHeaderPageBL()
+        mainPageBL.getHeaderPageBL()
                 .clickOnMyAccountButton()
                 .clickOnRegisterButton()
-                .registerNewPerson();
+                .registerNewPerson()
+                .verifyUserRegistration();
         randomEmail = RandomEmailUtil.email;
-        registerPageBL.verifyUserRegistration();
+        mainPageBL.getHeaderPageBL()
+                .clickOnMyAccountButton()
+                .clickOnLogoutButton()
+                .clickContinue()
+                .verifyLogout();
```

Comment:
You do not need this code, you already have this logic into AfterMethod()

Reply:
Remove this code.